contracts/base/Base.sol

```diff
@@ -67,6 +67,16 @@ contract Base is Initializable, ReentrancyGuard {
         _;
     }
 
+    /**
+        @notice Checks whether a given address is allowed (see Settings#hasPauserRole function) or not.
+        @dev It throws a require error if address is not allowed.
+        @param anAddress account to test.
+     */
+    modifier whenAllowed(address anAddress) {
+        require(_isAllowed(anAddress), "ADDRESS_ISNT_ALLOWED");
+        _;
+    }
+
```

Comment:
seems redundant to have an additional modifier and function that do the same thing as preexisting modifiers.
See other comments as well.

Reply:
Fixed